qiita_core/support_files/config_test.cfg

```diff
@@ -68,6 +68,12 @@ COOKIE_SECRET = SECRET
 # The value used to secure JWTs for delegated permission artifact download.
 JWT_SECRET = SUPER_SECRET
 
+# Address a user should write to when asking for help
+HELP_EMAIL = qiita.help@gmail.com
+
+# The email address, Qiita sends internal notifications to a sys admin
+SYSADMIN_EMAIL = jdereus@health.ucsd.edu
+
```

Comment:
Should these match the ones used in the actual test? 
```
# Address a user should write to when asking for help
HELP_EMAIL = foo@bar.com
# The email address, Qiita sends internal notifications to a sys admin
SYSADMIN_EMAIL = jeff@bar.com
```
and perhaps changing the error (which should never happen now) to a warning when the emails match those.